test/test-moc.js

```diff
@@ -156,7 +156,7 @@ assert.deepStrictEqual(Motoko.check("bad.mo"), {
 });
 
 const astString = JSON.stringify(
-  Motoko.parseMotoko(Motoko.readFile("ast.mo"))
+  Motoko.parseMotoko(/*enable_recovery=*/false, Motoko.readFile("ast.mo"))
```

Comment:
You should rebase considering https://github.com/serokell/motoko/pull/45/files, so you'll be able to add one test with and one without error recovery.

Reply:
done